paas_app_charmer/django/charm.py

```diff
@@ -44,6 +44,27 @@ def allowed_hosts_to_list(cls, value: str | None) -> typing.List[str]:
             return []
         return [h.strip() for h in value.split(",")]
 
+    @model_validator(mode="before")
+    @classmethod
+    def secret_key_id(cls, data: dict[str, str | int | bool | dict[str, str] | None]) -> dict:
+        """Read the new *-secret-key-id style configuration.
```

Comment:
"new" in docstrings like this tends to age poorly :)

Reply:
Updated, thanks!